requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyYAML
+pathlib
```

Comment:
I did a quick check on a RHEL6 host (rcm-pub-worker-01) and I don't see any pathlib installed or installable (`yum search pathlib`).  Are you aware whether this is obtainable for RHEL6 without extra steps?

If not then I think this dep should be avoided.  We dropped the idea of using only standard library here, but we're still constrained by the target platforms, one of which is RHEL6 / python2.6.

Reply:
I see it is available in epel6 - I'd still recommend against using it.  I know epel repo isn't enabled by default on at least some of the hosts where we'd want to deploy this, so this is potentially disruptive.